x/nodes/types/valUtil.go

```diff
@@ -42,7 +43,8 @@ func MustMarshalValidator(cdc *codec.Codec, validator Validator) []byte {
 func MustUnmarshalValidator(cdc *codec.Codec, valBytes []byte) Validator {
 	validator, err := UnmarshalValidator(cdc, valBytes)
 	if err != nil {
-		panic(err)
+		fmt.Println("Cannot unmarshal validator with bytes: ", hex.EncodeToString(valBytes))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("cannot unmarshal validator with bytes: %s", hex.EncodeToString(valBytes))
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)